migrations/versions/201506051421_4074211727ba_add_indico_unaccent_function.py

```diff
@@ -0,0 +1,37 @@
+"""ADD indico_unaccent function
+
+Revision ID: 4074211727ba
+Revises: 3f3a9554a6da
+Create Date: 2015-06-05 14:21:52.752777
+"""
+
+from indico.core.db.sqlalchemy.core import db
```

Comment:
Don't use `db` in an Alembic migration (besides that, you'd import it from `indico.core.db`). Instead, use the functions provided by alembic (check other migrations that need to execute raw SQL for examples).